docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_3.md

```diff
@@ -0,0 +1,57 @@
+- Non-admin user that has write rights in the folder
+
+**Preparations**:
+1. Login as the non-admin user from the prerequisites 
+2. Create object storage `storage3`
+3. In the `storage3`
+    - Create `folder1` with 2 files (`inner_file1` and `inner_file2`)
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the non-admin user from the prerequisites. Open the **Library** page | |
+| 2 | Open the object storage `storage3` created at step 2 of the Preparations | |
+| 3 | Click **+ Create v** button | |
+| 4 | Click on ***File*** item | |
+| 5 | In an appeared pop-up window specify a valid file name (`file1`), input text into the field _Content_ | |
+| 6 | Click **OK** button | New file appears in the table |
+| 7 | Logout and _Check audit log_ (see below) | Record <br> `WRITE storage3/file1` <br> is shown in the top of logs records |
+| 8 | Repeat steps 1-2 | |
+| 9 | Click **Upload** button | |
+| 10 | In pop-up window select a file (`file2`) and confirm uploading | |
+| 11 | Logout and _Check audit log_ (see below) | Record <br> `WRITE storage3/file2` <br> is shown in the top of logs records |
+| 12 | Repeat steps 1-2 | |
+| 13 | Click **Rename** button opposite `file1` file | |
+| 14 | Specify a new value in the _Name_ field. Click **OK** button | |
+| 15 | Logout and _Check audit log_ (see below) | Records <br> `READ storage3/file1` <br> `WRITE storage3/file1_new` <br> `DELETE storage3/file1` <br> are shown in the top of logs records | 
+| 16 | Repeat steps 1-2 | |
+| 17 | Click on `file2` name. Edit file content. Click "Save" button | |
+| 18 | Logout and _Check audit log_ (see below) | Records <br> `READ storage3/file2` (3 records) <br> `WRITE storage3/file2` <br> are shown in the top of logs records | 
+| 19 | Repeat steps 1-2 | |
+| 20 | Click **Rename** button opposite `folder1` folder | |
+| 21 | Specify a new value in the _Name_ field. Click **OK** button | |
+| 22 | Logout and _Check audit log_ (see below) | For each files from `folder1` (*inner_file1* and *inner_file2*) records <br> `READ storage3/folder1/<file>` <br> `WRITE storage3/folder1_new/<file>` <br> `DELETE storage3/folder1/<file>` <br> are shown in the top of logs records | 
+| 23 | Repeat steps 1-2 | |
+| 24 | Click **Remove** button near the `file2` name and confirm removing | |
+| 25 | Logout and _Check audit log_ (see below) | Record <br> `DELETE storage3/file2` <br> is shown in the top of logs records |
+| 26 | Repeat steps 1-2 | |
+| 27 | Click **Remove** button near the `folder1` name and confirm removing | |
+| 28 | Logout and _Check audit log_ (see below) | For each files from `folder1` (*inner_file1* and *inner_file2*) records <br> `DELETE storage3/folder1/<file>` <br> are shown in the top of logs records |
+| 29 | Repeat steps 1-2 | |
+| 30 | Click **Download** button near the `file1` name | |
+| 31 | Logout and _Check audit log_ (see below) | Record <br> `READ storage3/file1` <br> is shown in the top of logs records |
+
```

Comment:
Add the section **After** in which the storage created at the Preparations shall be removed.

Reply:
Done